esp-metadata-generated/src/_generated_esp32s2.rs

```diff
@@ -467,7 +467,8 @@ macro_rules! for_each_peripheral {
         _for_each_inner!((GPIO40 <= virtual())); _for_each_inner!((GPIO41 <= virtual()));
         _for_each_inner!((GPIO42 <= virtual())); _for_each_inner!((GPIO43 <= virtual()));
         _for_each_inner!((GPIO44 <= virtual())); _for_each_inner!((GPIO45 <= virtual()));
-        _for_each_inner!((GPIO46 <= virtual())); _for_each_inner!((AES <= AES()
+        _for_each_inner!((GPIO46 <= virtual())); _for_each_inner!((AES <= AES(AES : {
```

Comment:
Should these files be marked as "generated" in `.gitattributes` to have them automatically hidden? Dunno if people actually review the changes in these files.

Reply:
I don't like hiding changes, and here there is a real chance of something going wrong.